src/wireviz/Harness.py

```diff
@@ -20,16 +20,16 @@ def __init__(self):
         self.cables = {}
         self.additional_bom_items = []
 
-    def add_connector(self, name, *args, **kwargs):
+    def add_connector(self, name: str, *args, **kwargs) -> None:
         self.connectors[name] = Connector(name, *args, **kwargs)
 
-    def add_cable(self, name, *args, **kwargs):
+    def add_cable(self, name: str, *args, **kwargs) -> None:
         self.cables[name] = Cable(name, *args, **kwargs)
 
-    def add_bom_item(self, item):
+    def add_bom_item(self, item: str) -> None:
```

Comment:
The item is a `dict`, not a `str`.

Reply:
Thanks.  I haven't used this directly yet.  Fixed in most recent push.